src/main/java/com/dev/cinema/model/ShoppingCart.java

```diff
@@ -0,0 +1,53 @@
+package com.dev.cinema.model;
+
+import java.util.List;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.MapsId;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "shopping_carts")
+public class ShoppingCart {
+    @Id
+    @Column(name = "cart_id")
+    private Long id;
+    @OneToMany
+    @JoinTable(name = "tickets_cart",
+            joinColumns = @JoinColumn(name = "cart_id"),
+            inverseJoinColumns = @JoinColumn(name = "ticket_id"))
```

Comment:
Is that necessary since you're using join in your query?

Reply:
tried many different ways)))
refactor, thanks